adam/language/lexicon.py

```diff
@@ -27,6 +27,7 @@ class LexiconEntry:
     properties: ImmutableSet["LexiconProperty"] = attrib(
         converter=_to_immutableset, default=immutableset()
     )
+    plural_form: str = attrib(validator=instance_of(str), default="", kw_only=True)
```

Comment:
Prefer to make the potential absence of something explicit rather than using special values.

In this case, make the type of this `Optional[str]` with a default of `None`. While you are at it, change `verb_form_3SG_PRS` to be the same.

Reply:
Done